code/wdb/common.q

```diff
@@ -0,0 +1,528 @@
+/-Definitions for commonly/used parameters & functions in the .wdb namespace.
```

Comment:
I'm assuming nothing in this file has actually changed from it's original definition in `processes/wdb.q`?